Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -1148,7 +1148,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json">
-      <Version>12.0.2</Version>
+      <Version>13.0.0</Version>
```

Comment:
It looks like some places we use 13.0.1 and some 13.0.0. I think VS updated and then reverted versions.. can you update these to the latest version? I think the latest VS has 13.0.0 again but you should double check.